readme.md

```diff
@@ -89,6 +89,13 @@ Values: `'undocked'` `'right'` `'bottom'` `'previous'` `'detach'`
 
 The dock state to open DevTools in.
 
+##### preloads
+
+Type: `(string) => string[]`\
+Default: `(devtronPreloadFile) => [devtronPreloadFile]`
+
+Factory to provide scripts to set via `session#setPreloads`.
```

Comment:
This needs to explain why the option exists and why the user should not call `setPreloads` themselves.

Reply:
Actually, I think it would be better to do it the other way around and expose our preload thing, so the user could include it if they call `setPreloads` themselves.

Reply:
Maybe the naming should be more general too, in case we can to include other things in the preload script in the future.

Reply:
That seems better indeed. I exposed the resolved absolute file path and described the usage.
I renamed it to a more general "preload.js" but kept the explicit behaviour in the docs.